recipes/sicor/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "sicor" %}
+{% set version = "0.14.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sicor-{{ version }}.tar.gz
+  sha256: e790c84b6b139a6e13a9a8bafe5969e28f4c047634d01848b9f202434c2f2fab
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Can you add `python >=3.6` on build and run?
There is no python_requires upstream but the metadata indicates 3.6 is the minimum: https://gitext.gfz-potsdam.de/EnMAP/sicor/-/blob/master/setup.py#L81